packages/bundler-plugin-core/src/utils/getPreSignedURL.ts

```diff
@@ -65,8 +96,8 @@ export const getPreSignedURL = async ({
   let response: Response;
   try {
     response = await fetchWithRetry({
-      url,
-      retryCount,
+      url: `${apiUrl}${API_ENDPOINT}`,
+      retryCount: retryCount ?? DEFAULT_RETRY_COUNT,
```

Comment:
I would prefer if `DEFAULT_RETRY_COUNT` is an implemention detail of `fetchWithRetry`.

Reply:
ouu noice